.babelrc.json

```diff
@@ -0,0 +1,14 @@
+{
+  "presets": [
+    [
+      "@babel/env",
+      {
+        "modules": false,
+        "useBuiltIns": "usage",
+        "corejs": 3
+      }
+    ],
+    "@babel/typescript"
+  ],
+  "plugins": ["@babel/plugin-transform-modules-commonjs"]
```

Comment:
This can be removed once all `module.exports = foo` are eventually refactored out.